PostFreeSurfer/PostFreeSurferPipelineLongPrep.sh

```diff
@@ -406,19 +406,24 @@ if (( TemplateProcessing ==  1 )); then
         Timepoint_brain_mask_MNI="${AtlasSpaceFolder_timepoint}"/"$T1wImageBrainMask".nii.gz
 
         #link altas transforms to the timepoint directory.
-        if [ -d "$AtlasSpaceFolder_timepoint/xfms" ]; then 
+        if [ -d "$AtlasSpaceFolder_timepoint/xfms" ]; then
             rm -rf "$AtlasSpaceFolder_timepoint/xfms"
-        fi                
-        ln -sf "${AtlasSpaceFolder_template}/xfms" "${AtlasSpaceFolder_timepoint}/xfms"
+        fi
+        mkdir -p ${AtlasSpaceFolder_timepoint}
+        ln -sf "${AtlasSpaceFolder_template}/xfms" "${AtlasSpaceFolder_timepoint}/"
```

Comment:
This edit is already in master, including it in this PR is causing it to have a merge conflict.